runtime/oti/j9nonbuilder.h

```diff
@@ -4816,7 +4816,6 @@ typedef struct J9VMThread {
 	UDATA debugEventData6;
 	UDATA debugEventData7;
 	UDATA debugEventData8;
-	struct J9Class* methodHandlesLookupAccessClass;
```

Comment:
@gacholio Which parts of the J9VMThread are alignment-constrained?  Do you see an issue with this change?

Reply:
Anywhere the structure is ifdeffed is an alignment point. Given this is a UDATA field, it's probably OK here, but why would it not be on the end of the structure to avoid this question entirely?